compiler/plugins/target/ROCM/ROCMTarget.cpp

```diff
@@ -280,6 +280,16 @@ class ROCMTargetBackend final : public TargetBackend {
         subgroupSize = *setSubgroupSize;
       }
 
+      int64_t wavesPerEu = options.wavesPerEu;
+      IREE::Codegen::TranslationInfoAttr translationInfo =
+          getTranslationInfo(exportOp);
+      if (auto translationConfig = translationInfo.getConfiguration()) {
+        if (auto attr = dyn_cast_or_null<IntegerAttr>(
+                translationConfig.get("waves-per-eu"))) {
```

Comment:
nit: Can we keep it as "amdgpu-waves-per-eu" to match the llvmgpu config option name?

Reply:
keeping it consistent with llvm because this flag only makes sense for rocm.